lib/atoms/button/example.html

```diff
@@ -0,0 +1 @@
+<button id="{{id}}" class="{{class}}" ng-click="{{button}}">{{text}}</button>
```

Comment:
ng-click needs an expression (like a function) and we are not able to pass that from patterns.json configuration. I think we can put something like this on the example: ng-click="alert('Hello!')"

My mistake, sorry for not giving you more details.